apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1151,6 +1151,327 @@ describe("getSchedule", () => {
       expect(availableSlotsInTz.filter((slot) => slot.format().startsWith(plus2DateString)).length).toBe(23); // 2 booking per day as limit, only one booking on that
     });
 
+    test("test that booking limit is working correctly if user is all day available and attendee is in different timezone than host", async () => {
```

Comment:
test for fix of existing bug: https://github.com/calcom/cal.com/pull/16614/files#diff-3bcd8b09138a03622b300b58836dfed0598874aed41ff92f57c2b2700f30f4b5R284